### ISSUE TITLE
configurable keyboard shortcuts: labels are not localized in the editor (application settings)

### ISSUE THREAD

**Issue by danielweck** [edrlab/thorium-reader#967]
![Thorium_key_labels](https://user-images.githubusercontent.com/625201/75769947-ea640380-5d3e-11ea-9eaf-ff437956a61b.png)

**Comment by gautierchomel** [comment]
Would improve usability :)

Also the Edit button in front of each Shortcut should have a better accessible Name (like `aria-label="Edit Search Shortcut"`). In context Edit is sufficient, but while listing Buttons of the page we get a long list of similar Edit buttons.
 
![thorium-editer-shortcuts](https://user-images.githubusercontent.com/107039498/179302357-f9ce6ad2-b765-48c5-adee-c018583ce915.png)

**Comment by danielweck** [comment]
This is an advanced feature in Thorium's settings panel, users have not complained about missing translations because this is a really niche option occasionally used by people who understand the keyboard shortcuts organisation based on the Wiki documentation. Let's close this as it is unlikely to gain enough traction to justify the implementation effort, at this juncture.
